app/src/main/java/org/mozilla/focus/notification/BrowsingNotificationService.java

```diff
@@ -97,6 +97,10 @@ public int onStartCommand(Intent intent, int flags, int startId) {
                     activityIntent.setFlags(Intent.FLAG_ACTIVITY_NO_ANIMATION);
                 }
 
+                // This doesn't seem to be needed on Android 7.1. It is needed on Android 6, or otherwise
+                // we crash with "AndroidRuntimeException: Calling startActivity() from outside of an Activity  context requires the FLAG_ACTIVITY_NEW_TASK flag."
```

Comment:
Heh, the double spaces came from Android and I never noticed while copying and pasting : )